src/fides/api/migrations/hash_migration_mixin.py

```diff
@@ -0,0 +1,82 @@
+from abc import abstractmethod
+from typing import Any, List, Optional, Tuple
+
+from sqlalchemy import Boolean, Column, Index
+from sqlalchemy.ext.declarative import declared_attr
+
+from fides.api.migrations.hash_migration_tracker import HashMigrationTracker
+from fides.api.schemas.redis_cache import MultiValue
+
+
+class HashMigrationMixin:
+    """
+    A mixin that includes common logic for models that have hashed fields that need be be migrated from using bcrypt to SHA-256.
+    """
+
+    # set existing rows to false but set to true on create since we'll
+    # be using a new hashing function moving forward
+    is_hash_migrated = Column(Boolean, nullable=False, server_default="f", default=True)
+
+    @declared_attr
+    def __table_args__(cls: Any) -> Tuple:
```

Comment:
I was struggling with the type for `cls` so I just went with `Any`. Open to suggestions here!

Reply:
🤷 i wouldn't lose too much sleep over it